ch08/lambda/ingest-with-python/tests/test_suite.py

```diff
@@ -0,0 +1,69 @@
+#!/usr/bin/env python3
+
+from lambda_function import lambda_handler, schema
+
+from deltalake import DeltaTable
+import json
+import os
+import pyarrow as pa
+import pytest
+
+@pytest.fixture
```

Comment:
+1000. Thanks for including "how to test" in the examples. This is usually the biggest stumbling block when folk are getting up to speed. They can follow along, but then "what to do when things get real".